src/db/queries/transactionGroup.ts

```diff
@@ -360,10 +360,107 @@ export default function useTransactionGroup() {
     }
   }
 
+
+  const getTransactionsByGroupId = async ({ id }: { id: number}) => {
```

Comment:
```suggestion
  const get = async ({ id }: { id: number}) => {
```

um dem naming scheme _von mir_ zu entsprechen, weil wir die funktion ueber einen hook useTransactionGroup() callen ist es dann im context logisch von woher die get() funktion kommt und wir koennen es dann lokal umbenennen